### DISCUSSION TITLE
Reduce repetition for writing tests

### DISCUSSION THREAD

**Discussion by sunyu0410** [OSIPI/DCE-DSC-MRI_CodeCollection#117] [Ideas]
Here are some ideas to reduce repetition
1. Use *fixtures* to set up the input data for all test. Use `conftest.py` to share that across multiple files
2. Use test classes to similar tests

**Comment by mjt320** [comment]
Sounds sensible to me @sunyu0410
I don't have experience with fixtures or test classes - it will probably all make sense once there's an example to look at
